pjsip/src/pjsua-lib/pjsua_media.c

```diff
@@ -3983,9 +3983,7 @@ pj_status_t pjsua_media_channel_update(pjsua_call_id call_id,
 	    }
 
 	    /* Check if no media is active */
-	    if ((local_sdp->media[mi]->desc.port == 0) || 
-	        (si->dir == PJMEDIA_DIR_NONE))	    
-	    {
+	    if (local_sdp->media[mi]->desc.port == 0) {
```

Comment:
This change was introduce in https://github.com/pjsip/pjproject/issues/1899. However it is implemented for audio only. I think we should handle the same way for video.

Reply:
Good point. What do you think, @nanangizz ?

Reply:
Agree, I think the video should be treated the same way (in `port=0` vs `a=inactive` scenarios).